cmd/src/extensions.go

```diff
@@ -10,7 +10,8 @@ var extensionsCommands commander
 func init() {
 	usage := `'src extensions' is a tool that manages extensions in the extension registry on a Sourcegraph instance.
 
-EXPERIMENTAL: Extensions are experimental functionality on Sourcegraph and in the 'src' tool.
+DEPRECATED: We're in the process of removing Sourcegraph extensions with our September release.
+            Learn more: https://docs.sourcegraph.com/extensions/deprecation
```

Comment:
cc @jjinnii @muratsu @ryankscott

Reply:
Do you want this to go out in a 3.43.x version?

Reply:
I _think_ we do, since we are also shipping this already on sourcegraph.com but this is a questions for the above people to answer 🙈

Reply:
Ok, just let me know when you have a decision, since I already cut 3.43.0 for the branch cut this evening, but can cut 3.43.1 right after merge.

Reply:
Ah, I see. I don't think it's super urgent to get this out with 3.43 (the more important messaging is going out on the web app anyway), so I don't think we need to cut a minor release for that 🙂